client_code/model.py

```diff
@@ -32,4 +32,4 @@ class Vote:
     def __init__(self, ballot, proof, selection=None):
         self.ballot = ballot
         self.proof = proof
-        self.selection = selection
+        self.selection = selection
```

Comment:
this should not have been removed. `self.uuid` should be removed as it is equivalent to _ballotId